smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/DefaultHTTPProtocolCustomizations.kt

```diff
@@ -5,11 +5,15 @@
 
 package software.amazon.smithy.swift.codegen.integration
 
+import software.amazon.smithy.codegen.core.Symbol
 import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.model.traits.TimestampFormatTrait
 import software.amazon.smithy.swift.codegen.AuthSchemeResolverGenerator
+import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
+import software.amazon.smithy.swift.codegen.SmithyTestUtilTypes
 import software.amazon.smithy.swift.codegen.SwiftWriter
 
-abstract class DefaultHttpProtocolCustomizations : HttpProtocolCustomizable {
+abstract class DefaultHTTPProtocolCustomizations : HTTPProtocolCustomizable {
```

Comment:
Renamed and added some customization points that used to be passed piecemeal between codegen components.